packages/docs/src/styles/_examples.scss

```diff
@@ -213,6 +213,12 @@ $options-margin: $pt-grid-size * 3;
   animation: docs-wiggle-rotate $pt-transition-duration $pt-transition-ease infinite;
 }
 
+.docs-loading-skeleton-example-box {
+  flex: 1;
+  margin-right: $pt-grid-size * 1.5;
+  height: $pt-grid-size * 15;
```

Comment:
This line adds unnecessary extra space at the bottom of the card:
![image](https://cloud.githubusercontent.com/assets/199754/20983975/5419b3da-bc73-11e6-8dce-edbd0b24ff16.png)